### ISSUE TITLE
IoU Loss

### ISSUE THREAD

**Issue by LukeWood** [keras-team/keras-cv#113]
IoU loss is a simple, easy to understand bounding box loss function

Myself or Scott will probably write this to set the example for loss functions

**Comment by bhack** [comment]
https://github.com/tensorflow/models/blob/master/official/vision/beta/projects/yolo/losses/yolo_loss.py

**Comment by quantumalaviya** [comment]
@LukeWood I can work on this loss as well!

**Comment by abhi-glitchhg** [comment]
Hey there!
Any plans to add Complete IOU loss (CIOU) and Distance IOU loss (DIOU) from this paper -  https://arxiv.org/abs/1911.08287

cc : @LukeWood

**Comment by quantumalaviya** [comment]
Hey @abhi-glitchhg, I am planning to add these losses soon.

**Comment by LukeWood** [comment]
@quantumalaviya added this
